exercises/luhn/test/test_luhn.c

```diff
@@ -0,0 +1,109 @@
+#include "vendor/unity.h"
+#include "../src/luhn.h"
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+void test_single_digit_strings_can_not_be_valid(void)
+{
+   TEST_ASSERT_FALSE(luhn("1"));
+}
+
+void test_a_single_zero_is_invalid(void)
+{
+   TEST_IGNORE();               // delete this line to run test
+   TEST_ASSERT_FALSE(luhn("0"));
+}
+
+void test_a_simple_valid_SIN_that_remains_valid_if_reversed(void)
+{
+   TEST_IGNORE();
+   TEST_ASSERT_TRUE(luhn("059"));
+}
+
+void test_a_simple_valid_SIN_that_becomes_invalid_if_reversed(void)
+{
+   TEST_IGNORE();
+   TEST_ASSERT_TRUE(luhn("59"));
+}
+
+void test_a_valid_Canadian_SIN(void)
+{
+   TEST_IGNORE();
+   TEST_ASSERT_TRUE(luhn("055 444 285"));
```

Comment:
Have I misunderstood something? How does this test pass if spaces cause the function to exit early and return false?

Reply:
@wolf99 Spaces are just skipped. ```false``` is returned from the loop if the current char is not digit and also not ```' '``` (space). ```isspace``` checks for much more, which I think should be an error. For example this:
```
123<tab>   <spaces>456\r\n
<tab>        789\r
``` 
would be considered okay by using ```isspace```.

Reply:
Yes, this was my point. The readme doesn't specify `' '` only "space", which is somewhat subjective... only a suggestion anyway. Otherwise everything LGTM